packages/cascara/src/ui/Pagination/Pagination.js

```diff
@@ -78,7 +78,7 @@ const Pagination = ({ as, totalRecordCount, state }) => {
 
   return (
     <Boundaries>
-      <Role as={as} className={styles._}>
+      <Role {...rest} as={as} className={styles._}>
```

Comment:
```suggestion
      <Role {...rest} as={as} className={cx('_', rest.className)}>
```
We are muting classNames here

Reply:
Thank you for this suggestion. That helped to understand deeper :)